src/app/home/pages/PlansPage/components/Wizard/WizardComponent.tsx

```diff
@@ -276,8 +276,6 @@ const WizardComponent = (props: IOtherProps) => {
           title="Create a migration plan"
           onClose={handleClose}
           steps={steps}
-          isFullWidth
```

Comment:
@mturley Any alternative for this prop? Looks like it went away with pf 4.x

Reply:
It looks like it was just removed without an alternative, I assume because it's now the default behavior? I'll look into it.

Reply:
@ibolton336 I figured it out. Wizard was modified so that it can be used directly in a page layout or wrapped in a Modal. It still automatically renders in a modal when `isOpen` is passed for those who just need the default Modal props, but you can pull it out and wrap it in your own modal and pass a `width` to that modal. Here's a fix: https://github.com/mturley/mig-ui/commit/e386a3b9204a3b8d3978c2d50a448a393e3957c6

Reply:
Thanks mike - looks much better 👍